commands/build.go

```diff
@@ -41,6 +42,7 @@ func init() {
 	buildCmd.Flags().BoolVar(&shrinkwrap, "shrinkwrap", false, "Just write files to ./build/ folder for shrink-wrapping")
 	buildCmd.Flags().StringArrayVarP(&buildArgs, "build-arg", "b", []string{}, "Add a build-arg for Docker (KEY=VALUE)")
 	buildCmd.Flags().StringArrayVarP(&buildOptions, "build-option", "o", []string{}, "Set a build option, e.g. dev")
+	buildCmd.Flags().BoolVar(&tagsha, "tag-sha", false, "Tag function image with gitsha. This doesn't replase latest tag, but provides second with versioning")
```

Comment:
`tag-sha` was my suggestion, but I don't know if it's a good option. Do you have other suggestions?

`--auto-tag` or just `--tag` ? If users use this a lot we need it to be easy to use. Also You mention `gitsha` and I think you mean "Git SHA".

The comment `replase` is a typo. Let's just dump the second sentence.

Reply:
I would suggest `--auto-tag` as it's more descriptive.